frontend/test/metabase-visual/dashboard/parameters-widget.cy.spec.js

```diff
@@ -1,10 +1,14 @@
-import { restore, visitDashboard } from "__support__/e2e/helpers";
+import {
+  restore,
+  visitDashboard,
+  editDashboard,
+} from "__support__/e2e/helpers";
 import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
 
 const { PRODUCTS_ID } = SAMPLE_DATABASE;
 
 const questionDetails = {
-  query: { "source-table": PRODUCTS_ID },
+  query: { "source-table": PRODUCTS_ID, limit: 2 },
```

Comment:
This is not directly related to a flake, but it definitely doesn't hurt the overall performance.

When testing locally with the simulated CPU throttling (4x slowdown), tests were timing out because the initial dashboard results loading was too slow!

As a general rule, whenever you can limit the number of results without affecting the quality of the test, do it.

Reply:
⚡